Mage/Module.cs

```diff
@@ -67,28 +83,46 @@ public void SetupStaticData(Action<Exception> cb)
 		}
 
 
-		//
-		protected virtual string CommandPrefix
-		{
-			get { return null; }
-		}
+		// The module name as defined on the remote MAGE server
+		protected abstract string CommandPrefix { get; }
```

Comment:
Same as above, for me this is a choice of how we implement, either I'm fine with.